aspnet/fundamentals/logging.rst

```diff
@@ -146,6 +146,8 @@ To configure logging in your ASP.NET Core application, you should resolve ``ILog
 
 Once you've added ``ILoggerFactory`` as a parameter, you configure loggers within the ``Configure`` method by calling methods (or extension methods) on the logger factory. We have already seen an example of this configuration at the beginning of this article, when we added console logging by calling ``loggerFactory.AddConsole``.
 
+.. note:: You can optionally configure logging when setting up :doc:`hosting`, rather than in ``Startup``.
```

Comment:
We don't seem to have a section in hosting illustrating the use of the API `ConfigureLogging` or `UseLoggerFactory`.